lib/block-supports/typography.php

```diff
@@ -108,7 +108,12 @@ function gutenberg_apply_typography_support( $block_type, $block_attributes ) {
 	if ( $has_font_size_support && ! $should_skip_font_size ) {
 		$preset_font_size                    = array_key_exists( 'fontSize', $block_attributes ) ? "var:preset|font-size|{$block_attributes['fontSize']}" : null;
 		$custom_font_size                    = isset( $block_attributes['style']['typography']['fontSize'] ) ? $block_attributes['style']['typography']['fontSize'] : null;
-		$typography_block_styles['fontSize'] = $preset_font_size ? $preset_font_size : $custom_font_size;
+		$typography_block_styles['fontSize'] = $preset_font_size ? $preset_font_size : gutenberg_get_typography_font_size_value(
+			array(
+				'size'  => $custom_font_size,
+				'fluid' => true,
```

Comment:
Ah, we don't actually need this line since `gutenberg_get_typography_font_size_value()` checks explicitly for `false`
```suggestion
```

Reply:
Yes, good find! Let's ditch it (I think I got a bit confused when I encountered that earlier)